from unittest.mock import MagicMock


def mock_requests_get(url, **_):
    variant = url.split('/')[-1]
    return MockResponse(data_for_variant[variant])


class MockResponse(MagicMock):
    def __init__(self, data, **kwargs):
        super(MockResponse, self).__init__(data, **kwargs)
        self._data = data

    def json(self):
        return [self._data]

    @property
    def ok(self):
        if 'error' not in self._data:
            return True

    @property
    def status_code(self):
        if 'error' in self._data:
            return 400
        return 200

    @property
    def text(self):
        if 'error' in self._data:
            return self._data['error']


data_for_variant = {
    'NC_000001.11:g.215674515G>A': {'strand': 1, 'seq_region_name': '1', 'transcript_consequences': [{'cds_end': 13396, 'protein_end': 4466, 'sift_score': 0.18, 'polyphen_prediction': 'benign', 'sift_prediction': 'tolerated', 'cdna_start': 13835, 'hgnc_id': 'HGNC:12601', 'strand': -1, 'consequence_terms': ['missense_variant'], 'codons': 'Cct/Tct', 'gene_symbol_source': 'HGNC', 'transcript_id': 'ENST00000307340', 'impact': 'MODERATE', 'biotype': 'protein_coding', 'cds_start': 13396, 'cdna_end': 13835, 'gene_symbol': 'USH2A', 'variant_allele': 'A', 'gene_id': 'ENSG00000042781', 'protein_start': 4466, 'amino_acids': 'P/S', 'polyphen_score': 0.056}, {'polyphen_score': 0.018, 'gene_symbol': 'USH2A', 'gene_id': 'ENSG00000042781', 'variant_allele': 'A', 'amino_acids': 'P/S', 'protein_start': 4466, 'cdna_end': 13835, 'biotype': 'protein_coding', 'cds_start': 13396, 'transcript_id': 'ENST00000674083', 'impact': 'MODERATE', 'gene_symbol_source': 'HGNC', 'codons': 'Cct/Tct', 'consequence_terms': ['missense_variant'], 'polyphen_prediction': 'benign', 'sift_prediction': 'tolerated', 'cdna_start': 13835, 'hgnc_id': 'HGNC:12601', 'strand': -1, 'sift_score': 0.19, 'protein_end': 4466, 'cds_end': 13396}], 'allele_string': 'G/A', 'end': 215674515, 'input': 'NC_000001.11:g.215674515G>A', 'start': 215674515, 'most_severe_consequence': 'missense_variant', 'assembly_name': 'GRCh38', 'id': 'NC_000001.11:g.215674515G>A', 'colocated_variants': [{'allele_string': 'HGMD_MUTATION', 'end': 215674515, 'seq_region_name': '1', 'strand': 1, 'phenotype_or_disease': 1, 'id': 'CM172440', 'start': 215674515}, {'pubmed': [24033266, 28041643], 'var_synonyms': {'ClinVar': ['RCV001272942', 'RCV000672135', 'RCV001035021', 'RCV001004775', 'RCV000505044', 'RCV000041740', 'VCV000048417']}, 'start': 215674515, 'id': 'rs138398671', 'phenotype_or_disease': 1, 'clin_sig_allele': 'A:uncertain_significance;A:likely_pathogenic;A:likely_benign', 'seq_region_name': '1', 'strand': 1, 'clin_sig': ['uncertain_significance', 'likely_benign', 'likely_pathogenic'], 'end': 215674515, 'frequencies': {'A': {'gnomadg_afr': 4.826e-05, 'gnomade_fin': 4.619e-05, 'gnomadg_oth': 0.000478, 'gnomade_eas': 0, 'gnomadg_eas': 0, 'gnomade_oth': 0, 'gnomadg_amr': 0.0001308, 'gnomade_amr': 8.678e-05, 'gnomadg_sas': 0, 'gnomadg_ami': 0, 'gnomade_sas': 0, 'gnomadg': 0.0002497, 'gnomade_nfe': 0.0004348, 'gnomadg_asj': 0, 'gnomade_asj': 0, 'gnomadg_fin': 9.414e-05, 'gnomade': 0.0002157, 'gnomadg_nfe': 0.0004704, 'gnomade_afr': 6.16e-05, 'gnomadg_mid': 0}}, 'allele_string': 'G/A'}]},
    'NC_000001.11:g.40819893G>A': {'transcript_consequences': [{'variant_allele': 'A', 'strand': 1, 'biotype': 'protein_coding', 'cds_end': 853, 'impact': 'MODERATE', 'cdna_end': 1160, 'consequence_terms': ['missense_variant'], 'protein_end': 285, 'cdna_start': 1160, 'gene_symbol': 'KCNQ4', 'gene_symbol_source': 'HGNC', 'sift_score': 0, 'transcript_id': 'ENST00000347132', 'gene_id': 'ENSG00000117013', 'protein_start': 285, 'amino_acids': 'G/S', 'polyphen_score': 1, 'cds_start': 853, 'polyphen_prediction': 'probably_damaging', 'codons': 'Ggc/Agc', 'sift_prediction': 'deleterious', 'hgnc_id': 'HGNC:6298'}, {'cdna_start': 539, 'gene_symbol': 'KCNQ4', 'gene_symbol_source': 'HGNC', 'cdna_end': 539, 'impact': 'MODERATE', 'consequence_terms': ['missense_variant'], 'protein_end': 181, 'variant_allele': 'A', 'strand': 1, 'biotype': 'protein_coding', 'cds_end': 541, 'flags': ['cds_start_NF'], 'codons': 'Ggc/Agc', 'hgnc_id': 'HGNC:6298', 'sift_prediction': 'deleterious', 'amino_acids': 'G/S', 'polyphen_score': 1, 'cds_start': 541, 'polyphen_prediction': 'probably_damaging', 'protein_start': 181, 'sift_score': 0, 'transcript_id': 'ENST00000443478', 'gene_id': 'ENSG00000117013'}, {'cdna_end': 172, 'impact': 'MODIFIER', 'consequence_terms': ['non_coding_transcript_exon_variant'], 'variant_allele': 'A', 'gene_id': 'ENSG00000117013', 'transcript_id': 'ENST00000506017', 'strand': 1, 'biotype': 'processed_transcript', 'hgnc_id': 'HGNC:6298', 'cdna_start': 172, 'gene_symbol': 'KCNQ4', 'gene_symbol_source': 'HGNC'}, {'hgnc_id': 'HGNC:6298', 'sift_prediction': 'deleterious', 'codons': 'Ggc/Agc', 'amino_acids': 'G/S', 'polyphen_prediction': 'probably_damaging', 'polyphen_score': 0.995, 'cds_start': 853, 'protein_start': 285, 'transcript_id': 'ENST00000509682', 'gene_id': 'ENSG00000117013', 'sift_score': 0, 'gene_symbol': 'KCNQ4', 'gene_symbol_source': 'HGNC', 'cdna_start': 853, 'protein_end': 285, 'cdna_end': 853, 'impact': 'MODERATE', 'consequence_terms': ['missense_variant'], 'strand': 1, 'biotype': 'protein_coding', 'cds_end': 853, 'variant_allele': 'A'}], 'colocated_variants': [{'start': 40819893, 'strand': 1, 'phenotype_or_disease': 1, 'allele_string': 'HGMD_MUTATION', 'id': 'CM990769', 'seq_region_name': '1', 'end': 40819893}, {'allele_string': 'HGMD_MUTATION', 'id': 'CM990770', 'seq_region_name': '1', 'end': 40819893, 'start': 40819893, 'strand': 1, 'phenotype_or_disease': 1}, {'allele_string': 'COSMIC_MUTATION', 'seq_region_name': '1', 'id': 'COSV61273409', 'end': 40819893, 'somatic': 1, 'strand': 1, 'start': 40819893, 'var_synonyms': {'COSMIC': ['COSM6691713']}, 'phenotype_or_disease': 1}, {'phenotype_or_disease': 1, 'clin_sig': ['pathogenic'], 'var_synonyms': {'OMIM': [603537.0001, 603537.0004], 'UniProt': ['VAR_001547'], 'ClinVar': ['RCV000006622', 'VCV000006244', 'RCV000006619', 'VCV000006241', 'RCV000844633', 'RCV000211722']}, 'start': 40819893, 'strand': 1, 'clin_sig_allele': 'T:pathogenic;A:pathogenic', 'end': 40819893, 'id': 'rs28937588', 'pubmed': [24033266, 20301388, 8035838, 10369879, 20966080, 25116015, 10025409, 18786918, 30311386], 'seq_region_name': '1', 'allele_string': 'G/A/T'}], 'allele_string': 'G/A', 'input': 'NC_000001.11:g.40819893G>A', 'regulatory_feature_consequences': [{'biotype': 'promoter', 'variant_allele': 'A', 'consequence_terms': ['regulatory_region_variant'], 'regulatory_feature_id': 'ENSR00001167503', 'impact': 'MODIFIER'}], 'id': 'NC_000001.11:g.40819893G>A', 'assembly_name': 'GRCh38', 'start': 40819893, 'strand': 1, 'end': 40819893, 'seq_region_name': '1', 'most_severe_consequence': 'missense_variant'},
    'NC_000001.11:g.40819893T>A': {'error': "Unable to parse HGVS notation \'NC_000001.11:g.40819893T>A\': Reference allele extracted from NC_000001.11:40819893-40819893 (G) does not match reference allele given by HGVS notation NC_000001.11:g.40819893T>A (T)"},
    'NC_000002.12:g.39006443C>T': {'id': 'NC_000002.12:g.39006443C>T', 'most_severe_consequence': 'synonymous_variant', 'colocated_variants': [{'seq_region_name': '2', 'var_synonyms': {'ClinVar': ['RCV000522430', 'RCV001138302', 'RCV001138301', 'RCV000038537', 'VCV000040708', 'RCV001813297']}, 'strand': 1, 'pubmed': [25741868, 24033266], 'phenotype_or_disease': 1, 'allele_string': 'C/T', 'frequencies': {'T': {'gnomade_nfe': 0.0001853, 'gnomadg_mid': 0, 'gnomadg': 0.0004932, 'gnomade_eas': 0, 'gnomadg_afr': 0, 'gnomadg_sas': 0, 'gnomade_sas': 0, 'gnomadg_fin': 0, 'gnomadg_oth': 0, 'gnomadg_amr': 6.558e-05, 'gnomade_afr': 0, 'gnomade': 0.0008772, 'gnomade_amr': 0.0005788, 'gnomadg_eas': 0, 'gnomadg_nfe': 0.0001323, 'gnomade_fin': 0, 'gnomade_asj': 0.01719, 'gnomadg_asj': 0.01873, 'gnomadg_ami': 0, 'gnomade_oth': 0.0009823}}, 'clin_sig_allele': 'T:uncertain_significance;T:benign;T:benign/likely_benign', 'start': 39006443, 'id': 'rs149092581', 'end': 39006443, 'clin_sig': ['uncertain_significance', 'benign', 'likely_benign']}], 'start': 39006443, 'end': 39006443, 'input': 'NC_000002.12:g.39006443C>T', 'seq_region_name': '2', 'assembly_name': 'GRCh38', 'allele_string': 'C/T', 'strand': 1, 'transcript_consequences': [{'protein_start': 920, 'protein_end': 920, 'codons': 'agG/agA', 'gene_symbol_source': 'HGNC', 'strand': -1, 'amino_acids': 'R', 'consequence_terms': ['synonymous_variant'], 'variant_allele': 'T', 'biotype': 'protein_coding', 'impact': 'LOW', 'cdna_end': 2788, 'gene_symbol': 'SOS1', 'hgnc_id': 'HGNC:11187', 'transcript_id': 'ENST00000395038', 'cds_end': 2760, 'cds_start': 2760, 'gene_id': 'ENSG00000115904', 'cdna_start': 2788}, {'impact': 'LOW', 'gene_symbol': 'SOS1', 'cdna_end': 3389, 'hgnc_id': 'HGNC:11187', 'transcript_id': 'ENST00000402219', 'cds_end': 2760, 'cds_start': 2760, 'gene_id': 'ENSG00000115904', 'cdna_start': 3389, 'protein_start': 920, 'protein_end': 920, 'codons': 'agG/agA', 'gene_symbol_source': 'HGNC', 'strand': -1, 'amino_acids': 'R', 'consequence_terms': ['synonymous_variant'], 'variant_allele': 'T', 'biotype': 'protein_coding'}, {'biotype': 'retained_intron', 'variant_allele': 'T', 'gene_id': 'ENSG00000115904', 'cdna_start': 556, 'consequence_terms': ['non_coding_transcript_exon_variant'], 'transcript_id': 'ENST00000474390', 'strand': -1, 'hgnc_id': 'HGNC:11187', 'gene_symbol_source': 'HGNC', 'cdna_end': 556, 'gene_symbol': 'SOS1', 'impact': 'MODIFIER'}, {'hgnc_id': 'HGNC:11187', 'gene_symbol': 'SOS1', 'cdna_end': 1594, 'impact': 'LOW', 'cdna_start': 1594, 'gene_id': 'ENSG00000115904', 'transcript_id': 'ENST00000685279', 'cds_start': 1527, 'cds_end': 1527, 'gene_symbol_source': 'HGNC', 'protein_end': 509, 'codons': 'agG/agA', 'protein_start': 509, 'biotype': 'protein_coding', 'variant_allele': 'T', 'consequence_terms': ['synonymous_variant'], 'strand': -1, 'amino_acids': 'R'}, {'hgnc_id': 'HGNC:11187', 'gene_symbol_source': 'HGNC', 'distance': 4507, 'gene_symbol': 'SOS1', 'impact': 'MODIFIER', 'biotype': 'retained_intron', 'variant_allele': 'T', 'consequence_terms': ['downstream_gene_variant'], 'gene_id': 'ENSG00000115904', 'transcript_id': 'ENST00000688043', 'strand': -1}, {'gene_symbol': 'SOS1', 'cdna_end': 2767, 'impact': 'MODIFIER', 'gene_symbol_source': 'HGNC', 'hgnc_id': 'HGNC:11187', 'transcript_id': 'ENST00000689668', 'strand': -1, 'biotype': 'retained_intron', 'variant_allele': 'T', 'consequence_terms': ['non_coding_transcript_exon_variant'], 'gene_id': 'ENSG00000115904', 'cdna_start': 2767}, {'amino_acids': 'R', 'strand': -1, 'biotype': 'nonsense_mediated_decay', 'variant_allele': 'T', 'consequence_terms': ['synonymous_variant', 'NMD_transcript_variant'], 'protein_start': 883, 'gene_symbol_source': 'HGNC', 'codons': 'agG/agA', 'protein_end': 883, 'cds_start': 2649, 'transcript_id': 'ENST00000690876', 'cds_end': 2649, 'cdna_start': 2666, 'gene_id': 'ENSG00000115904', 'cdna_end': 2666, 'gene_symbol': 'SOS1', 'impact': 'LOW', 'hgnc_id': 'HGNC:11187'}, {'protein_end': 843, 'codons': 'agG/agA', 'gene_symbol_source': 'HGNC', 'protein_start': 843, 'consequence_terms': ['synonymous_variant'], 'variant_allele': 'T', 'biotype': 'protein_coding', 'amino_acids': 'R', 'strand': -1, 'hgnc_id': 'HGNC:11187', 'impact': 'LOW', 'cdna_end': 2546, 'gene_symbol': 'SOS1', 'gene_id': 'ENSG00000115904', 'cdna_start': 2546, 'transcript_id': 'ENST00000691229', 'cds_start': 2529, 'cds_end': 2529}, {'cds_start': 2649, 'transcript_id': 'ENST00000692089', 'cds_end': 2649, 'cdna_start': 2666, 'gene_id': 'ENSG00000115904', 'gene_symbol': 'SOS1', 'cdna_end': 2666, 'impact': 'LOW', 'hgnc_id': 'HGNC:11187', 'amino_acids': 'R', 'strand': -1, 'biotype': 'protein_coding', 'variant_allele': 'T', 'consequence_terms': ['synonymous_variant'], 'protein_start': 883, 'gene_symbol_source': 'HGNC', 'protein_end': 883, 'codons': 'agG/agA'}, {'protein_start': 152, 'gene_symbol_source': 'HGNC', 'codons': 'agG/agA', 'protein_end': 152, 'amino_acids': 'R', 'strand': -1, 'variant_allele': 'T', 'biotype': 'protein_coding', 'consequence_terms': ['synonymous_variant'], 'gene_symbol': 'SOS1', 'cdna_end': 456, 'flags': ['cds_start_NF'], 'impact': 'LOW', 'hgnc_id': 'HGNC:11187', 'transcript_id': 'ENST00000692227', 'cds_start': 456, 'cds_end': 456, 'cdna_start': 456, 'gene_id': 'ENSG00000115904'}, {'cdna_end': 1359, 'gene_symbol': 'SOS1', 'impact': 'MODIFIER', 'gene_symbol_source': 'HGNC', 'hgnc_id': 'HGNC:11187', 'transcript_id': 'ENST00000692620', 'strand': -1, 'biotype': 'nonsense_mediated_decay', 'variant_allele': 'T', 'cdna_start': 1359, 'gene_id': 'ENSG00000115904', 'consequence_terms': ['3_prime_UTR_variant', 'NMD_transcript_variant']}]},
    'NC_000006.12:g.152387156G>A': {'seq_region_name': '6', 'start': 152387156, 'input': 'NC_000006.12:g.152387156G>A', 'allele_string': 'G/A', 'transcript_consequences': [{'biotype': 'protein_coding', 'amino_acids': 'Y', 'cds_start': 8403, 'variant_allele': 'A', 'impact': 'LOW', 'cdna_end': 8968, 'gene_symbol': 'SYNE1', 'gene_symbol_source': 'HGNC', 'codons': 'taC/taT', 'gene_id': 'ENSG00000131018', 'cdna_start': 8968, 'strand': -1, 'cds_end': 8403, 'transcript_id': 'ENST00000367255', 'protein_start': 2801, 'protein_end': 2801, 'hgnc_id': 'HGNC:17089', 'consequence_terms': ['synonymous_variant']}, {'cds_start': 8424, 'amino_acids': 'Y', 'variant_allele': 'A', 'impact': 'LOW', 'gene_symbol': 'SYNE1', 'cdna_end': 8900, 'biotype': 'protein_coding', 'transcript_id': 'ENST00000423061', 'protein_start': 2808, 'protein_end': 2808, 'consequence_terms': ['synonymous_variant'], 'hgnc_id': 'HGNC:17089', 'gene_symbol_source': 'HGNC', 'codons': 'taC/taT', 'gene_id': 'ENSG00000131018', 'cdna_start': 8900, 'strand': -1, 'cds_end': 8424}, {'transcript_id': 'ENST00000454018', 'variant_allele': 'A', 'impact': 'MODIFIER', 'gene_symbol': 'SYNE1', 'hgnc_id': 'HGNC:17089', 'consequence_terms': ['upstream_gene_variant'], 'gene_symbol_source': 'HGNC', 'flags': ['cds_start_NF'], 'biotype': 'protein_coding', 'distance': 1480, 'gene_id': 'ENSG00000131018', 'strand': -1}, {'strand': -1, 'cdna_start': 8621, 'gene_id': 'ENSG00000131018', 'biotype': 'retained_intron', 'gene_symbol_source': 'HGNC', 'consequence_terms': ['non_coding_transcript_exon_variant'], 'gene_symbol': 'SYNE1', 'cdna_end': 8621, 'hgnc_id': 'HGNC:17089', 'impact': 'MODIFIER', 'variant_allele': 'A', 'transcript_id': 'ENST00000461872'}], 'colocated_variants': [{'var_synonyms': {'COSMIC': ['COSM288577']}, 'seq_region_name': '6', 'end': 152387156, 'somatic': 1, 'phenotype_or_disease': 1, 'strand': 1, 'start': 152387156, 'id': 'COSV54898013', 'allele_string': 'COSMIC_MUTATION'}, {'start': 152387156, 'end': 152387156, 'phenotype_or_disease': 1, 'strand': 1, 'seq_region_name': '6', 'clin_sig': ['uncertain_significance', 'likely_benign'], 'var_synonyms': {'PhenCode': ['SYNE1:c.8403C>T'], 'ClinVar': ['RCV001412128', 'RCV000724249', 'RCV000252722', 'VCV000197975']}, 'pubmed': [25741868], 'allele_string': 'G/A', 'id': 'rs757104773', 'frequencies': {'A': {'gnomade_eas': 0, 'gnomade_asj': 9.932e-05, 'gnomade_sas': 0.0002613, 'gnomadg_nfe': 0.0001176, 'gnomadg_asj': 0, 'gnomade': 0.0001076, 'gnomadg_fin': 0, 'gnomadg_amr': 0, 'gnomadg_oth': 0.000478, 'gnomadg': 8.546e-05, 'gnomade_fin': 0, 'gnomade_oth': 0.0001634, 'gnomade_amr': 0, 'gnomadg_ami': 0, 'gnomade_afr': 0, 'gnomade_nfe': 0.0001501, 'gnomadg_mid': 0, 'gnomadg_afr': 2.414e-05, 'gnomadg_eas': 0, 'gnomadg_sas': 0.0006214}}, 'clin_sig_allele': 'A:likely_benign;A:conflicting_interpretations_of_pathogenicity'}], 'most_severe_consequence': 'synonymous_variant', 'regulatory_feature_consequences': [{'impact': 'MODIFIER', 'variant_allele': 'A', 'biotype': 'open_chromatin_region', 'consequence_terms': ['regulatory_region_variant'], 'regulatory_feature_id': 'ENSR00000809941'}], 'end': 152387156, 'assembly_name': 'GRCh38', 'id': 'NC_000006.12:g.152387156G>A', 'strand': 1},
    'NC_000001.11:g.215674515G>A': {'input': 'NC_000001.11:g.215674515G>A', 'colocated_variants': [{'end': 215674515, 'start': 215674515, 'id': 'CM172440', 'allele_string': 'HGMD_MUTATION', 'strand': 1, 'phenotype_or_disease': 1, 'seq_region_name': '1'}, {'clin_sig_allele': 'A:uncertain_significance;A:likely_pathogenic;A:likely_benign', 'allele_string': 'G/A', 'var_synonyms': {'ClinVar': ['RCV001272942', 'RCV000672135', 'RCV001035021', 'RCV001004775', 'RCV000505044', 'RCV000041740', 'VCV000048417']}, 'clin_sig': ['uncertain_significance', 'likely_benign', 'likely_pathogenic'], 'id': 'rs138398671', 'start': 215674515, 'end': 215674515, 'seq_region_name': '1', 'phenotype_or_disease': 1, 'pubmed': [24033266, 28041643], 'strand': 1, 'frequencies': {'A': {'gnomadg': 0.0002497, 'gnomadg_asj': 0, 'gnomadg_eas': 0, 'gnomade_amr': 8.678e-05, 'gnomade_fin': 4.619e-05, 'gnomade_asj': 0, 'gnomade_nfe': 0.0004348, 'gnomadg_oth': 0.000478, 'gnomadg_ami': 0, 'gnomadg_amr': 0.0001308, 'gnomadg_afr': 4.826e-05, 'gnomade_sas': 0, 'gnomadg_sas': 0, 'gnomadg_mid': 0, 'gnomade_oth': 0, 'gnomadg_fin': 9.414e-05, 'gnomadg_nfe': 0.0004704, 'gnomade_eas': 0, 'gnomade': 0.0002157, 'gnomade_afr': 6.16e-05}}}], 'start': 215674515, 'allele_string': 'G/A', 'assembly_name': 'GRCh38', 'end': 215674515, 'strand': 1, 'transcript_consequences': [{'amino_acids': 'P/S', 'protein_start': 4466, 'strand': -1, 'polyphen_prediction': 'benign', 'protein_end': 4466, 'polyphen_score': 0.056, 'cdna_end': 13835, 'cdna_start': 13835, 'gene_symbol_source': 'HGNC', 'impact': 'MODERATE', 'transcript_id': 'ENST00000307340', 'sift_prediction': 'tolerated', 'gene_id': 'ENSG00000042781', 'gene_symbol': 'USH2A', 'codons': 'Cct/Tct', 'sift_score': 0.18, 'hgnc_id': 'HGNC:12601', 'biotype': 'protein_coding', 'cds_start': 13396, 'variant_allele': 'A', 'cds_end': 13396, 'consequence_terms': ['missense_variant']}, {'transcript_id': 'ENST00000674083', 'impact': 'MODERATE', 'sift_prediction': 'tolerated', 'gene_symbol_source': 'HGNC', 'cdna_start': 13835, 'cdna_end': 13835, 'polyphen_score': 0.018, 'protein_end': 4466, 'polyphen_prediction': 'benign', 'protein_start': 4466, 'strand': -1, 'amino_acids': 'P/S', 'consequence_terms': ['missense_variant'], 'cds_end': 13396, 'variant_allele': 'A', 'cds_start': 13396, 'hgnc_id': 'HGNC:12601', 'biotype': 'protein_coding', 'sift_score': 0.19, 'codons': 'Cct/Tct', 'gene_symbol': 'USH2A', 'gene_id': 'ENSG00000042781'}], 'id': 'NC_000001.11:g.215674515G>A', 'seq_region_name': '1', 'most_severe_consequence': 'missense_variant'},
}
